apps/roam/scripts/compile.ts

```diff
@@ -170,9 +170,26 @@ export const compile = ({
       },
     }),
   );
+
+  if (process.env.NODE_ENV === "production") {
+    console.log("Saving to root for build.sh publishing to Roam Depot");
+    const distFiles = fs.readdirSync(outdir);
+    distFiles.forEach((file) => {
+      const srcPath = path.join(outdir, file);
+      const destPath = path.join(root, file);
+      if (!fs.existsSync(destPath)) fs.cpSync(srcPath, destPath);
+    });
+  }
```

Comment:
This PR did two main things:
- add this code
- add `build.ts` to call `compile`, for a better naming convention and to set `NODE_ENV` to `prod`